pkg/storageos/deploy.go

```diff
@@ -35,7 +35,7 @@ const (
 	statefulsetKind = "statefulset"
 	deploymentKind  = "deployment"
 
-	daemonsetName   = "storageos-daemonset"
+	DaemonSetName   = "storageos-daemonset"
```

Comment:
Out of curiosity why does this var need to be public now?

Reply:
It's being used in the e2e test for this change.